src/groupInfo/components/GroupGalleryMap.vue

```diff
@@ -61,15 +66,7 @@ export default {
       if (this.myCoordinates) {
         coordsForBounds.push(this.myCoordinates)
       }
-      const bounds = L.latLngBounds(coordsForBounds).pad(0.2)
-      if (this.offset.lat !== 0 || this.offset.lng !== 0) {
```

Comment:
Just for my understanding: Was this the code that emulated the paddingTopLeft that is now fixed?